neo/Prompt/Commands/tests/test_claim_command.py

```diff
@@ -111,23 +114,23 @@ def test_3_wallet_no_claimable_gas(self):
 
         self.assertFalse(relayed)
 
-    def test_4_keyboard_interupt(self):
-        with patch('sys.stdout', new=StringIO()) as mock_print:
-            with patch('neo.Prompt.Commands.Wallet.prompt', side_effect=[KeyboardInterrupt]):
-                wallet = self.GetWallet1()
-
-                claim_tx, relayed = ClaimGas(wallet)
-                self.assertEqual(claim_tx, None)
-                self.assertFalse(relayed)
-                self.assertIn("Claim transaction cancelled", mock_print.getvalue())
```

Comment:
Why was this test deleted?

Reply:
I'm guessing same reason as the wallet cancellation. Added it back.